moja-kuhinjica-frontend/src/pages/mealReservation/index.tsx

```diff
@@ -204,6 +212,14 @@ const MealReservation = (): JSX.Element => {
         setConfirmationModalIsOpen(false)
     }
 
+    const isBookingAllowed = (): boolean => {
+        const cutoffHour = 10
```

Comment:
cutOffHour = 10; 
better naming time before order
also need to be on top file const